### ISSUE TITLE
Syntax error in dialects/n+obsd/dproc.c

### ISSUE THREAD

**Issue by albert-github** [lsof-org/lsof#138]
In the file: dialects/n+obsd/dproc.c on line 189 we see the construct:
```
#endif	/* defined(HASKVMGETPROC2) &/
```
I don't think this is intended and now the code till the end of the next comment is ignored (and even creating a nested comment, which is not allowed).
I think the line should read:
```
#endif	/* defined(HASKVMGETPROC2) */
```

**Comment by masatake** [comment]
Thank you but none maintains n+obsd dialect.
I will accept a pull request if you send it. In such a case, please use `[n+obsd]` as the prefix of the header of the commit log.

**Comment by albert-github** [comment]
Even tough the code is not maintained it should not have these kind of errors and they still should be corrected.
I'm not a user of the software but saw the problem "by accident" when walking through the doxygen log files on Fossies.org.

I've just pushed a proposed patch, pull request #140

**Comment by masatake** [comment]
Fixed in #140 
Thank you.
